ckanext/recombinant/controller.py

```diff
@@ -181,7 +181,42 @@ def template(self, dataset_type, lang, owner_org):
         except NotFound:
             abort(404, _('Not found'))
 
+
```

Comment:
Please add a docstring to the this method describing the POST parameters that this endpoint now accepts.